src/lib/components/ui/Tabs/README.mdx

```diff
@@ -0,0 +1,242 @@
+          isActive={activeTab === 1}
+          label="Design"
+          link={{
+            href: '#',
+            onClick: (e) => navigate(e, 1),
+          }}
+        />
+        <TabsItem
+          beforeLabel={<Icon icon="code" />}
+          isActive={activeTab === 2}
+          label="Code"
+          link={{
+            href: '#',
+            onClick: (e) => navigate(e, 2),
+          }}
+        />
+        <TabsItem
+          beforeLabel={<Icon icon="star" />}
+          isActive={activeTab === 3}
+          label="Resources"
+          link={{
+            href: '#',
+            onClick: (e) => navigate(e, 3),
+          }}
+        />
+      </Tabs>
+    );
+  }}
+</Playground>
+
+## Scrollable Tabs
+
+If you have more than a few tabs, you may need to make sure they will be all
+accessible no matter the space they have around. Wrap Tabs into
+[ScrollView](/components/ui/scroll-view) to make them scrollable if necessary.
+
+<Playground>
+  {() => {
+    const [activeTab, setActiveTab] = React.useState(1);
+    const navigate = (event, tab) => {
+      setActiveTab(tab);
+      event.preventDefault();
+    };
+    return (
+      <div style={{ width: '19rem' }}>
+        <ScrollView direction="horizontal">
+          <Tabs>
+            <TabsItem
+              isActive={activeTab === 1}
+              label="Design"
+              link={{
+                href: '#',
+                onClick: (e) => navigate(e, 1),
+              }}
+            />
+            <TabsItem
+              isActive={activeTab === 2}
+              label="Code"
+              link={{
+                href: '#',
+                onClick: (e) => navigate(e, 2),
+              }}
+            />
+            <TabsItem
+              isActive={activeTab === 3}
+              label="Resources"
+              link={{
+                href: '#',
+                onClick: (e) => navigate(e, 3),
+              }}
+            />
+            <TabsItem
+              isActive={activeTab === 4}
+              label="Other"
+              link={{
+                href: '#',
+                onClick: (e) => navigate(e, 4),
+              }}
+            />
+            <TabsItem
+              isActive={activeTab === 5}
+              label="More"
+              link={{
+                href: '#',
+                onClick: (e) => navigate(e, 5),
+              }}
+            />
+          </Tabs>
+        </ScrollView>
+      </div>
+    );
+  }}
+</Playground>
+
+## API
+
+<Props table of={Tabs} />
+
+### TabsItem
+
+<Props table of={TabsItem} />
+
+## Theming
+
+### Tabs Theming
+
+| Custom Property                                      | Description                                                  |
+|------------------------------------------------------|--------------------------------------------------------------|
+| `--rui-Tabs__border-width`                           | Width of decorative bottom border                            |
```

Comment:
You talk here about bottom border but property name is without "bottom" in it. Is it correct? Seems like a bug.